goat@.service

```diff
@@ -9,7 +9,7 @@ ConditionPathExists=/usr/bin/goat
 Type=oneshot
 User=root
 Group=root
-ExecStart=/usr/bin/goat "%i" --log-level=info
+ExecStart=/usr/bin/goat "%i"
```

Comment:
not related to this MR, but it might be cool to be able to set flags either using a config file or env variables.

Reply:
My opinion is that there aren't enough flags for that to be worth the implementation cost

Reply:
We can revisit later

Reply:
But in the context of a systemd unit I can see why it's more useful to set an env var or config file than to change the unit constantly and do `daemon reload`...